### ISSUE TITLE
Link Opens So Many Times

### ISSUE THREAD

**Issue by Menteroth** [PedroS11/twitch-live-extension#52]
Why does this link open so often when I start Chrome? It's very annoying.

https://github.com/PedroS11/twitch-live-extension/blob/master/changelogs/4_0_0.md

**Comment by PedroS11** [comment]
Hey, is that so? I've not facing that issue and it really shouldn't be like that because, if you look at this line (https://github.com/PedroS11/twitch-live-extension/blob/master/src/background/background.ts#L64), that link only opens when you install the extension or update to a newer version. Did you uninstall and install it multiple times? Once you update it, you shouldn't be getting it again

**Comment by Menteroth** [comment]
I only changed to this new version one time. So not sure what is happening. I mean, it's less than I made it out to be when I wrote this. It's a few times now and then. Just felt more since I've seen it quite a few times since the changes hehe.

**Comment by PedroS11** [comment]
And it didn't change anything in like 2 months, it was to be some bug with Chrome, I'll see if I find a cause for it, if not, I may have to remove that feature until they fix it, sorry about that :/

**Comment by Menteroth** [comment]
Could it be when Chrome browser is updated or something? Just thinking of anything here.
No need to be sorry hehe. It's all good. I was already annoyed with other stuff when I posted this so I exaggerated a bit. But it reopens sometimes at least so just wanted to let you know.

**Comment by PedroS11** [comment]
No problem at all, I didn't find anything related with the issue on the extension :D
Do you still have the issue?

**Comment by Menteroth** [comment]
Yeah. Weird thing though. I already had Chrome open and I clicked a Twitter link from Discord and then it opened the mentioned page haha. So not sure what's going on.

**Comment by PedroS11** [comment]
No worries, I think I found the reason, the fix should go live this week and thanks for letting know about it!
https://github.com/PedroS11/twitch-live-extension/compare/fix-tab-opens-too-many-times?expand=1

**Comment by PedroS11** [comment]
Just sent the new version for chrome approval, as soon as they allow me, I'll make it live

**Comment by Menteroth** [comment]
Oh cool that you actually might have found it hehe. Care to share? I probably won't understand too much since I don't code myself but still curious haha.

**Comment by PedroS11** [comment]
Basically, when chrome was updated, it would trigger my function to open that tab with the instructions, so now, if it's a chrome update, I skip it
You can look at it [here](https://github.com/PedroS11/twitch-live-extension/pull/54/files#diff-099ec6c6a55fde47bdf9be5a4207c64b21c0b143bbc7e0d43e04cd30bed788f2)

**Comment by PedroS11** [comment]
Yeah but it makes sense this time ahahah I made an update today (you can now search through your Favorites) so you'll see it opening, let me know if you get it again, you shouldn't see it hopefully

**Comment by Menteroth** [comment]
This update was like 5 minutes ago if that matters. But otherwise I'll check next one hehe. All good.

**Comment by PedroS11** [comment]
Yeah yeah, it matches the time I pushed the newest version, all good. Thanks for letting me know, if you get it again, let a message here so I can have another look at it
